common/test/acceptance/tests/helpers.py

```diff
@@ -225,6 +225,9 @@ def select_option(query, value):
         try:
             select = Select(query.first.results[0])
             select.select_by_visible_text(value)
+            # If the element is still present, focus out of the element
```

Comment:
This is a comment that was for a previous if condition that I had implemented that never got deleted, this can be removed.